src/test/resources/org/prebid/server/ApplicationTest/openrtb2/test-auction-request.json

```diff
@@ -506,7 +506,7 @@
         }
       },
       "cache": {
-        "bids": []
+        "bids": {}
```

Comment:
Add vastxml object and extend bids object with ttl parameter to check that json can be successfully parsed to ExtRequestPrebid and hb_uuid/hb_uuid_{BIDDER} keywords appears in response.